DQM/SiStripCommissioningSources/plugins/tracking/SiStripFineDelayHit.h

```diff
@@ -44,9 +45,9 @@ class SiStripFineDelayHit : public edm::EDProducer {
       virtual void beginRun(const edm::Run &, const edm::EventSetup &) override;
       virtual void produce(edm::Event&, const edm::EventSetup&) override;
       virtual void produceNoTracking(edm::Event&, const edm::EventSetup&);
-      std::pair<uint32_t, uint32_t> deviceMask(const StripSubdetector::SubDetector subdet,const int substructure);
-      std::vector< std::pair<uint32_t,std::pair<double, double> > > detId(const TrackerGeometry& tracker,const reco::Track* tk, const std::vector<Trajectory>& trajVec, const StripSubdetector::SubDetector subdet = StripSubdetector::TIB,const int substructure=0xff);
-      std::vector< std::pair<uint32_t,std::pair<double, double> > > detId(const TrackerGeometry& tracker,const reco::Track* tk, const std::vector<Trajectory>& trajVec, const uint32_t& maskDetId, const uint32_t& rootDetId);
+      std::pair<uint32_t, uint32_t> deviceMask(const StripSubdetector::SubDetector subdet,const int substructure,const TrackerTopology* tkrTopo);
```

Comment:
Shall we add a typedef `using DeviceMask = std::pair<uint32_t,uint32_t>;`, and use that consistently throughout? I think it would improve readability.